x-pack/plugins/apm/public/components/app/Home/index.tsx

```diff
@@ -68,16 +70,29 @@ function getHomeTabs({
       render: () => <ServiceMap />,
       name: 'service-map',
     });
+
+    homeTabs.push({
+      link: (
+        <RumOverviewLink>
+          {i18n.translate('xpack.apm.home.rumTabLabel', {
+            defaultMessage: 'Real User Monitoring',
+          })}
+        </RumOverviewLink>
+      ),
+      render: () => <RumOverview />,
+      name: 'rum-overview',
+    });
```

Comment:
I think you want to move this tab to `x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceDetailTabs.tsx` and make it conditional so it only shows up if `isRumAgentName`.

Reply:
i believe i don't need to make any change here after our offline discussion.